guides/v2.1/cloud/env/variables-deploy.md

```diff
@@ -106,6 +106,25 @@ stage:
       _merge: true
 ```
 
+### `ENABLE_GOOGLE_ANALYTICS`
+
+-  **Default**—_false_
+-  **Version**—Magento 2.1.4 and later
+
+```yaml
+stage:
+  deploy:
+   ENABLE_GOOGLE_ANALYTICS: false 
+```
+
+Enables or disables Google Analytics when deploying to Staging and Integration environments.
+
+-   **`true`**—Enables Google Analytics on Staging and Integration environments.
+-   **`false`**—Disables Google Analytics on Staging and Integration environments.
+
+{:.bs-callout .bs-callout-info}
+The {{ site.data.var.ece }} deploy process always enables Google Analytics when deploying to Production environments.
```

Comment:
This note seems slightly redundant, but it's not a show-stopper.

Reply:
The note just clarifies that this environment variable doesn't affect Production. I added it because of your comment on the dev PR about being clear about environments.

Reply:
By redundant, I was referring to indicating it is during deploy. The note itself is important to have.